src/features/newProtectedData/NewProtectedData.tsx

```diff
@@ -179,7 +179,7 @@ export default function NewProtectedData() {
             target="_blank"
             sx={{ color: 'green', textDecorationColor: 'green' }}
           >
-            View on iExec Explorer
+            You can see it here
```

Comment:
We should use a different text, because one may think that you can see the protected data by clicking the link. 

I suggest we use something like "View Details" or "View metadata"

Reply:
I will see with adoption. But it use that choose this wording